horde/apis/v2/base.py

```diff
@@ -1637,13 +1641,22 @@ class Models(Resource):
         required=False,
         default="all",
         type=str,
+        choices=MODEL_STATES,
         help=(
             "If 'known', only show stats for known models in the model reference. "
             "If 'custom' only show stats for custom models. "
             "If 'all' shows stats for all models."
         ),
         location="args",
     )
+    get_parser.add_argument(

```

Comment:
bool arguments in metadata are a bit buggy. A `?metadata=False` query is treated as `True`, sadly, because it stupidly casts the string into a bool. To work around this, I suggest no default and then treat None as "False". 

To properly work around this requires a bit of a hack by looking into the request headers

Sadly the library we're using doesn't seem maintained well anymore :(